cityofaustin/atd-moped
# adds missing columns to csv export

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10765

this pr adds all missing columns displayed in the project list view to the csv export. it also adds a few helper functions to properly display missing and more complex values that were displayed as `null` or `[Object Object]` in the csv. it does not strip out the html tags from the `project_note` column - please let me know if you think we should add a helper function to do so!

## Testing
**URL to test:** 

https://deploy-preview-891--atd-moped-main.netlify.app/

**Steps to test:**
1. go to project list view and click download to export a csv
2. open the csv
3. you should see a readable export of all columns
4. any missing values should be displayed as "-"
5. columns with multiple values should be displayed as comma separated lists

---
#### Ship list
